zepto.js

```diff
@@ -939,9 +936,6 @@
   return $
 })()
 
-window.Zepto = Zepto
```

Comment:
see @rayrutjes This was the culprit of all our issues. We will be able to remove every code where we try to be "smart" and show/hide the fact that we use zepto.